apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -226,29 +250,55 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 		return reconcile.Result{}, errGetSec
 	}
 
-	//get certificate
-	certificateSecret := &corev1.Secret{}
-	errc := r.client.Get(context.TODO(), types.NamespacedName{Name: security.Spec.Certificate, Namespace: wso2NameSpaceConst}, certificateSecret)
+	var certificateSecret = &corev1.Secret{}
+	var alias string
+	//keep to track the existance of certificates
+	var existcert bool
+	//to add multiple certs with alias
+	var certList map[string]string
+	var certName string
+	//get the volume mounts
+	jobVolumeMount, jobVolume := getVolumes(instance)
 
-	if errc != nil && errors.IsNotFound(errc) {
-		reqLogger.Info("defined cretificate is not found")
-		return reconcile.Result{}, errc
-	}
+	//get certificate for JWT and Oauth
+	if strings.EqualFold(security.Spec.Type, "Oauth") || strings.EqualFold(security.Spec.Type, "JWT") {
+
+		errc := r.client.Get(context.TODO(), types.NamespacedName{Name: security.Spec.Certificate, Namespace: wso2NameSpaceConst}, certificateSecret)
```

Comment:
The line length is exceeded. Shall we split this into new line?

Reply:
Resolved